vite.config.ts

```diff
@@ -6,9 +6,29 @@ import react from "@vitejs/plugin-react";
 import rescript from "@jihchi/vite-plugin-rescript";
 import { viteCommonjs } from "@originjs/vite-plugin-commonjs";
 
+// Needed to allow ending paths with a version number (e.g. /v1.0.0), if not
+// the server will try to find a file with that name and fail.
+const allowPathWithEndingVersionNumbers = () => ({
```

Comment:
Wouahou that must have been a pain to debug

Reply:
:no_mouth: